app/src/androidTest/java/woowacourse/movie/db/history/ReservationHistoryDaoTest.kt

```diff
@@ -0,0 +1,82 @@
+package woowacourse.movie.db.history
+
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Before
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.runner.RunWith
+import woowacourse.movie.TestFixture.makeMockTicket
+
+@RunWith(AndroidJUnit4::class)
+class ReservationHistoryDaoTest {
+    private lateinit var reservationDao: ReservationHistoryDao
+    private lateinit var db: ReservationHistoryDatabase
```

Comment:
```suggestion
@RunWith(AndroidJUnit4::class)
class ReservationHistoryDaoTest {
    private lateinit var reservationDao: ReservationHistoryDao
    private lateinit var testReservationDao: TestReservationDao
    private lateinit var db: ReservationHistoryDatabase
```

모킹을 할 필요없이 그냥 테스트 환경 세팅을 하는 역핢만을 담당하는 Dao 를 하나 새로 만들어 주면 되지 않나요?
제가 생각했던 TestReservationHistoryDao 는 ReservationHistoryDao 의 모킹이 아닌,
테스트 환경에서만 접근가능한 독립된 Dao 입니다.

```kt
@Dao
interface TestReservationDao {
    @Query("DELETE FROM reservationTicket")
    fun clearReservations()
}
```

Reply:
TestReservationDao로 기존 DB에 접근하는 방법을 고민하였습니다.
TestReservationDao.clearReservations()를 실행한다고 해도 로컬 DB에는 적용이 안될 것 같아서요
여러 방법을 찾아봤는데, 테스트를 위한 확장함수를 활용하는 방법을 적용해봤습니다.

Reply:
> 여러 방법을 찾아봤는데, 테스트를 위한 확장함수를 활용하는 방법을 적용해봤습니다.

오! 제가 원래 생각했던 방법보다 더 좋은 것 같습니다.
한 수 배워갑니다. 👍 💯